YAROD - Yet Another Round Of Docs (#3274)
tenzir/tenzir: web/docs/user-guides/export-from-a-node.md

# Export from a node

> Exporting (or *querying*) data is equivalent to [running a

Reviewer: Unrelated to your diff, but "equivalent" seems to be the wrong choice here.

```suggestion
Exporting (or *querying*) data can be done by [running a
```
